.github/workflows/test-kms.yml

```diff
@@ -0,0 +1,61 @@
+name: Run KMS tests
+
+on:
+  push:
+  workflow_dispatch:
+
+permissions: {}
+
+jobs:
+  test-kms:
+    runs-on: ubuntu-latest
+
+    permissions:
+      id-token: 'write' # for OIDC auth for GCP authentication
+      issues: 'write' # for filing an issue on failure
+
+    steps:
+      - name: Checkout securesystemslib
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
+
+      - name: Set up Python
+        uses: actions/setup-python@13ae5bb136fac2878aff31522b9efb785519f984
```

Comment:
ugh, I'm actually missing some required parameters here now that I use a current version I think. Will check tomorrow

Reply:
fixed now: https://github.com/jku/securesystemslib/actions/runs/3590694453/jobs/6044359440